spec/models/observation_spec.rb

```diff
@@ -3095,6 +3095,31 @@
       expect(o.identifications.count).to eq 2
     end
 
+    it "change should be triggered by activating a taxon" do
+      load_test_taxa
+      o = Observation.make!
+      i1 = Identification.make!( observation: o, taxon: @Pseudacris_regilla_regilla )
```

Comment:
If this test doesn't rely on the taxon being a subspecies, could you switch to using `@Pseudacris_regilla` and remove the additional taxon you're adding in `load_test_taxa`? `load_test_taxa` is really onerous and runs all over the place, so adding more work there will slow down a lot of tests. Or, if you only need a genus -> species -> subspecies taxonomy and not a full branch from subspecies down the root, it might be better to use the blueprints to set up the taxonomy you need in this test instead of using `load_test_taxa`

Reply:
sure - I was trying to replicate Jane's case - but I since learned that this isn't ssp specific so will change it